.travis.yml

```diff
@@ -1,24 +1,26 @@
 sudo: required
-
 language: c
-os: linux
-dist: trusty
-
+os:
+  - linux
 services:
   - docker
+env:
+  - PG_VERSION=9.6.5
+  - PG_VERSION=10.0
+
+before_install:
```

Comment:
I couldn't get it to work with a job stage combined with a build matrix using multiple env variables. Travis ran two "default" stages instead, simply trying to compile a standard C project with `./configure && make` (which is Travis' default script for C projects). The only way I found to override the default script was to move this top-level. Maybe Travis stages don't work well in combination with a build matrix.

If you know a way to get the build matrix to work with stages, I am happy to change.

Still, I am not sure why we should use stages at all since it just slows down the build by running things in serial fashion instead of parallel.

Reply:
hmm. I can look into this after the merge. Haven't tried the stages with a matrix. FYI, stages with the same name run in parallel. stages with different names run sequentially

Reply:
So, if stages have the same name and run in parallel, what is the point of using stages?